experimental/packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -348,6 +349,20 @@ export const getOutgoingRequestAttributes = (
   return Object.assign(attributes, options.hookAttributes);
 };
 
+/**
+ * Returns outgoing request Metric attributes scoped to the request data
+ * @param {SpanAttributes} spanAttributes the span attributes
+ */
+export const getOutgoingRequestMetricAttributes = (
+  spanAttributes: SpanAttributes
+): MetricAttributes => {
+  const metricAttributes: MetricAttributes = {};
+  metricAttributes[SemanticAttributes.HTTP_METHOD] = spanAttributes[SemanticAttributes.HTTP_METHOD];
+  metricAttributes[SemanticAttributes.NET_PEER_NAME] = spanAttributes[SemanticAttributes.NET_PEER_NAME];
+  metricAttributes[SemanticAttributes.HTTP_URL] = spanAttributes[SemanticAttributes.HTTP_URL];
```

Comment:
High cardinality is still an issue here and with target for server metrics, is there any utils that I can use to take care of this?, any pointers would be appreciated.

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#parameterized-attributes

Reply:
You should look into what i've done with https://github.com/open-telemetry/opentelemetry-js/pull/2208, its our mechanism for http frameworks that have the http "target" information but you'll need to modify a lot of code (which i think could be done in different PRs)

Reply:
Can't we introduce a hook that returns sanitised version of url? A user could then replace all numbers or UUID like strings to placeholders to lower the high cardinality

Reply:
> Can't we introduce a hook that returns sanitised version of url? A user could then replace all numbers or UUID like strings to placeholders to lower the high cardinality

It can but it might be complicated for users to configure this, we should aim for auto instrumentations that mostly works out of the box imo

Reply:
The spec requires this attribute value to be parameterized. If it is not parameterized, it should be omitted to prevent high cardinality.

Reply:
I think it would be better to tackle this in a seperate PR, i agree that we should avoid having high cardinality attribute in such important metric by default

Reply:
Removed attributes and added TODOs for both URL in client metric and target in server metric.